storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -384,15 +384,15 @@ RS_Status SetOperationPKCol(const NdbDictionary::Column *col, NdbOperation *oper
     const size_t encoded_str_len = request->PKValueLen(colIdx);
 
     // The buffer in out has been allocated by the caller and is at least 3/4 the size of the input.
-    const int maxEncodedSize = ((4 * KEY_MAX_SIZE_IN_BYTES_DECODED) / 3);
+    const int maxEncodedSize = ((4 * KEY_MAX_SIZE_IN_BYTES) / 3);
     if (unlikely(encoded_str_len > maxEncodedSize)) {
         return RS_CLIENT_ERROR(std::string(ERROR_008) +
                         " Encoded data length is greater than 4/3 of maximum binary size." +
                         " Column: " + std::string(col->getName()) +
-                        " Maximum binary size: " + std::to_string(KEY_MAX_SIZE_IN_BYTES_DECODED));
+                        " Maximum binary size: " + std::to_string(KEY_MAX_SIZE_IN_BYTES));
     }
 
-    char pk[KEY_MAX_SIZE_IN_BYTES_DECODED];
+    char pk[KEY_MAX_SIZE_IN_BYTES];
```

Comment:
this one (line 395) was not supposed to change. KEY_MAX_SIZE_IN_BYTES --> KEY_MAX_SIZE_IN_BYTES_DECODED